samples/glean/metrics.yaml

```diff
@@ -1,22 +1,33 @@
-# This file contains a definition of the scalar probes that are recorded in Telemetry.
-# They are submitted with the "main" pings and can be inspected in about:telemetry.
+# This file defines the metrics that are recorded by glean telemetry. They are
+# automatically converted to Kotlin code at build time using the `glean_parser`
+# PyPI package.
 
-# The following section contains the accessibility scalars.
-a11y:
-  instantiators:
-    bug_numbers:
-      - 1323069
-      - 1462238
+$schema: http://mozilla.org/schemas/glean/metrics
+
+browser_engagement:
+  click:
+    type: event
+    description: >
+      Just testing events
+    bugs:
+      - 123456789
+    notification_emails:
+      - telemetry-client-dev@mozilla.com
+    objects:
+      - object1
+      - object2
+    extra_keys:
+      key1: "This is key one"
+      key2: "This is key two"
+
+basic:
```

Comment:
Should this group end up in a separate file? This is not for this PR, but we should clarify the expected behaviour

Reply:
This does end up in a separate generated Kotlin file.  Are you thinking it should live in a separate (input) `metrics.yaml` as well?

Reply:
🤦‍♂️ sorry, my bad! No, I meant separate generated Kotlin files :)